core/sections/clause_6a_deprecated.adoc

```diff
@@ -0,0 +1,29 @@
+
+[[deprecated]]
+== Deprecated
+
+[[deprecated-maxOccurs]]
+=== `maxOccurs` parameter
+
+The <<ogc_process_description,"OGC Process Description">> requirements class makes use of the <<process-input-schema,`maxOccurs`>> parameter to set the maximum cardinality of a process input.  This standard deprecates the use of this parameter and implementors should be aware that it will be removed in subsequent versions of this document.
+
+Instead, setting the cardinality of an input beyond 1 should be done using a https://json-schema.org/draft/2020-12/json-schema-core#section-10.3.1[JSON-Schema array] with the https://json-schema.org/draft/2020-12/json-schema-validation#section-6.4.2[`minItems`] and https://json-schema.org/draft/2020-12/json-schema-validation#section-6.4.1[`maxItems`] parameters in the input's schema specified using the <<process-input-schema,schema>> member.
+
+Furthermore, the value of the `minOccurs` parameter is constrained to the values "0" or "1".
+
+Requirement <<req_core_process-execute-input-array,/req/core/process-execute-input-array>> is also deprecated.
+
+With reference to <<cardinality_rules>>, for implementors wishing to maintain backward compatability with the behaviour when `minOccurs` is "1" and `maxOccurs` is greater than "1", the following schema fragment should be used:
+
+[source,YAML]
+----
+schema:
+  oneOf:
+    - <original-schema>
+    - type: array
+      items: <original-schema>
+      minItems: 1
+      maxItems: N
+----
```

Comment:
One issue with this is that it will not be possible to distinguish whether this `schema` represents the previous 1.0 `minOccurs=1`/`maxOccurs=1` (which could be omitted), or the new (2.0?) strategy to represent cardinality.

Issue #373 doesn't seem resolved with this approach alone, but rather made more ambiguous, if one intends to be backward compatible. 

A process description using 2.0 might need to always report `minOccurs=1` and `maxOccurs=1` explicitly to disambiguate.

Reply:
I am updating this PR to undo the deprecation of minOccurs and maxOccurs.

Reply:
clause_6a_deprecated.adoc has been removed.